ui-components/src/blocks/ActionBlock.tsx

```diff
@@ -41,7 +41,7 @@ const ActionBlock = ({
     <div className={actionBlockClasses.join(" ")}>
       <div className="action-block__head">
         {icon && <div className="action-block__icon">{icon}</div>}
-        <h3 className="action-block__header">{header}</h3>
+        <h2 className="action-block__header">{header}</h2>
```

Comment:
Looked at how this ActionBlock component was used and it seemed ok to just change the tag here. Let me know if this should be configurable.